site2/docs/admin-api-persistent-topics.md

```diff
@@ -1,80 +1,78 @@
 ---
 id: admin-api-persistent-topics
-title: Managing persistent topics
-sidebar_label: Persistent topics
+title: Manage topics
+sidebar_label: Topics
 ---
 
-Persistent helps to access topic which is a logical endpoint for publishing and consuming messages. Producers publish messages to the topic and consumers subscribe to the topic, to consume messages published to the topic.
-
-In all of the instructions and commands below, the topic name structure is:
-
+Pulsar has persistent and non-persistent topics. Persistent topic is a logical endpoint for publishing and consuming messages. The topic name structure for persistent topics is:
 
 ```shell
 persistent://tenant/namespace/topic
 ```
 
-## Persistent topics resources
+Non-persistent topics are used in applications that only consume real-time published messages and do not need persistent guarantee. In this way, it reduces message-publish latency by removing overhead of persisting messages. The topic name structure for non-persistent topics is:
 
-### List of topics
+```shell
+non-persistent://tenant/namespace/topic
+```
+## Manage topic resources
+Whether it is persistent or non-persistent topic, you can obtain the topic resources through `pulsar-admin` tool, REST API and Java.
 
-It provides a list of persistent topics exist under a given namespace.
+### List of topics
 
-#### pulsar-admin
+You can get the list of topics under a given namespace in the following ways.
 
-List of topics can be fetched using [`list`](../../reference/CliTools#list) command.
+<!--DOCUSAURUS_CODE_TABS-->
+<!--pulsar-admin-->
 
 ```shell
-$ pulsar-admin persistent list \
+$ pulsar-admin topics list \
   my-tenant/my-namespace
 ```
 
-#### REST API
-
-{@inject: endpoint|GET|/admin/v2/persistent/:tenant/:namespace|operation/getList}
-
-#### Java
+<!--REST API-->
+{@inject: endpoint|GET|/admin/v2/topics/:tenant/:namespace|operation/getList}
```

Comment:
I don't think we have the `topics` endpoint to get all the topics. Actually, there have two steps on the client-side, the client will get the persistent topics and non-persistent topics then combine the result at the client-side.
I am not sure how to add two REST API endpoints. @tuteng Any ideas?

Reply:
Please refer to `NON-PERSISTENT TOPIC` http://pulsar.apache.org/admin-rest-api/?version=2.6.1&apiversion=v2#operation/getList and `PERSISTENT TOPIC` http://pulsar.apache.org/admin-rest-api/?version=2.6.1&apiversion=v2#operation/getList

Reply:
{@inject: endpoint|GET|/admin/v2/:schema/:tenant/:namespace|operation/getList}